drivers/extcon/extcon-sm5502.c

```diff
@@ -144,6 +144,9 @@ enum sm5502_muic_acc_type {
 	SM5502_MUIC_ADC_AUDIO_TYPE1_FULL_REMOTE = 0x3e,	/* |      001|11110| */
 	SM5502_MUIC_ADC_AUDIO_TYPE1_SEND_END = 0x5e,	/* |      010|11110| */
 							/* |Dev Type1|--ADC| */
+
+	SM5502_MUIC_ADC_GROUND_OTG = 0x80,		/* |      100|00000| */ /* FIXME */
```

Comment:
Hmm, why is this `FIXME`? Also maybe

```suggestion
	SM5502_MUIC_ADC_GROUND_USB_OTG = 0x80,		/* |      100|00000| */
```

for consistency with the other.

Reply:
I wasn't sure if I wrote those bits correctly. Will remove as well as newlines around

Reply:
looks fine to me, only upstream will know if it's good enough :P